### PR TITLE
Adds support for command packages

### PR DESCRIPTION
**Pull request by jsancio**
WIP. Was able to install the command.

### CODE REVIEW

**Comment by ConnorDoyle** [comment]
@jsancio is this in it's final shape / ready for review?

**Comment by jsancio** [comment]
@ConnorDoyle No, sorry. Let me finish the dcos cli changes first in case I have to change more stuff.

**Comment by ConnorDoyle** [comment]
OK, just making sure you weren't blocked on this.

**Comment by jsancio** [comment]
@ConnorDoyle it should be reviewable now. Thanks!

File: repo/meta/schema/command-schema.json [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
+          },
+          "title": "Embedded Requirements File",
+          "description": "An array of strings representing of the requirements file to use for installing the subcommand for Pip. Each item is interpreted as a line in the requirements file."
         }
-      }
-    ]
-  }
+      },
+      "additionalProperties": false,
+      "required": ["pip"]
```
Comment (ConnorDoyle) [inline repo/meta/schema/command-schema.json]: Should this be required?  Other options (something other than pip) might be added later.

Comment (ConnorDoyle) [inline repo/meta/schema/command-schema.json]: Nevermind, I get it now :+1:

File: repo/packages/H/helloworld/0/package.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "helloworld",
+  "version": "0.1.0",
+  "website": "https://github.com/mesosphere/dcos-helloworld",
+  "maintainer": "support@mesosphere.io",
+  "description": "Example DCOS Subcommand",
+  "tags": [ "mesosphere", "example", "subcommand", "helloworld" ]
```
Comment (ConnorDoyle) [inline repo/packages/H/helloworld/0/package.json]: No need to duplicate the package name in these tags.

File: repo/packages/H/helloworld/0/package.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "helloworld",
+  "version": "0.1.0",
+  "website": "https://github.com/mesosphere/dcos-helloworld",
+  "maintainer": "support@mesosphere.io",
+  "description": "Example DCOS Subcommand",
```
Comment (ConnorDoyle) [inline repo/packages/H/helloworld/0/package.json]: This package is more than just a subcommand.  How about "Example DCOS application package"?

**Comment by ConnorDoyle** [comment]
Looks good!  Please rebase & :shipit: after looking at the comments above.

File: repo/packages/H/helloworld/0/package.json (hunk 3 above)
Comment (jsancio) [inline repo/packages/H/helloworld/0/package.json]: Done.

File: repo/packages/H/helloworld/0/package.json (hunk 2 above)
Comment (jsancio) [inline repo/packages/H/helloworld/0/package.json]: Done.

File: repo/meta/schema/command-schema.json (hunk 1 above)
Comment (jsancio) [inline repo/meta/schema/command-schema.json]: :+1:

**Comment by jsancio** [comment]
@ConnorDoyle updated the PR.

**Comment by ConnorDoyle** [comment]
One more rebase (sorry) and then let's land it.

**Pull request merged**